action.yml

```
@@ -65,6 +69,8 @@ runs:
         INPUT_GITHUB_TOKEN: ${{ inputs.github-token }}
         INPUT_MINOR_CONTRIBUTORS_LIMIT: ${{ inputs.minor-contributors-limit }}
         INPUT_SHOW_CONTRIBUTORS_LIMIT: ${{ inputs.show-contributors-limit }}
+        INPUT_SHOW_ALL_LANGUAGES: ${{ inputs.show-all-languages }}
+        INPUT_SHOW_ALL_LANGUAGES: ${{ inputs.show-all-languages }}
```

YAMLlint (1.37.1)

[error] duplication of key "INPUT_SHOW_ALL_LANGUAGES" in mapping

(key-duplicates)